lib/registries/addon/branded/new/template.hbs

```diff
@@ -29,41 +29,45 @@
                             {{t 'registries.new.step_one_project_heading'}}
                         </legend>
                         <div local-class='RadioButtonAndLabelDiv'>
-                            <input
-                                data-test-has-project-button
-                                local-class='RadioElement RadioInput'
-                                type='radio'
-                                name='HasProject'
-                                value='yes'
-                                onclick={{fn (mut this.isBasedOnProject) true}}
-                                checked={{this.isBasedOnProject}}
-                            >
-                            <label
-                                local-class='RadioElement RadioLabel {{if this.isBasedOnProject 'IsChecked'}}'
-                                for='HasProject'
-                            >
-                                {{t 'registries.new.yes'}}
-                            </label>
+                            {{#let (unique-id) 'has-project' as |hasProjectId|}}
```

Comment:
Out of curiosity, what led you to this solution? It is not something I'd heard of needing to do before.

Reply:
The `<label>` tag binds to its corresponding `<input>` by either matching the input's `id` and label's `for` (as shown here), or by nesting the `<input>` inside of the `<label>` (this is what was causing the weird behavior before). This is a pattern we used in the draft/metadata page to match the unique id's across these elements